configuration/pih/htmlforms/ncd-adult-followup.xml

```diff
@@ -709,7 +709,7 @@
 
                                 <includeIf velocityTest="$pt_has_rf or $pt_has_lf or $pt_has_hf">
                                     <tr>
-                                        <obsgroup groupingConceptId="PIH:12636">
+                                        <obsgroup groupingConceptId="PIH:12657">
```

Comment:
This isn't correct.  This is "Dietary intake" and not a class="ConvSet".  What are you trying to do here @tommyiversonj